src/backend/api/Fusion.Resources.Api/Controllers/Person/ApiModels/ApiPersonAbsence.cs

```diff
@@ -26,10 +26,31 @@ public ApiPersonAbsence(QueryPersonAbsence absence, bool hidePrivateNotes)
                 TaskDetails = (absence.TaskDetails != null) ? ApiTaskDetails.Hidden : null;
             }
         }
+        public ApiPersonAbsence(QueryPersonAbsenceBasic absence, bool hidePrivateNotes)
```

Comment:
Could this "business logic" be done outside of the constructor ?
In controller?

We did something similar when hiding proposals querying for allocation requests.